### DISCUSSION TITLE
Transitioning from Static to Dynamic Permission Groups

### DISCUSSION THREAD

**Discussion by mauroalexandre** [frangoteam/FUXA#1484] [Ideas]
Hi team,

Currently, our permission groups are static, but I believe we need to implement a dynamic registration system for them. This change could be somewhat complex as it impacts the entire system. I suggest the following steps for implementation:

1. **Initial Setup:** When creating the project in the project file, we should establish the existing groups to maintain backward compatibility.
   
2. **UI Integration:** Create a context menu within the user screen for registering new groups.
   
3. **Authorization Update:** Modify the authorization screen to recognize these new groups.

I believe these steps will help us transition smoothly and enhance our system's flexibility. Your feedback on this plan will be invaluable.

Looking forward to your insights.

**Comment by unocelli** [comment]
Hi Mauro, the feature is already present in the Pro version as Roles and therefore the only applicable solution is to import the code into the basic version as well. Please do not invest time on this feature. I will see to integrating it very soon.

**Reply by mauroalexandre** [reply]
Ah, nice!

**Comment by henjoe** [comment]
Just do it Bro so everybody will get happy 😁
